optional_plugins/html/MANIFEST.in

```diff
@@ -1,2 +1,2 @@
 recursive-include avocado_result_html/templates *
-include VERSION
\ No newline at end of file
+include VERSION README.rst
```

Comment:
new line missing

Reply:
Actually, this is how all the `MANIFEST.in` are currently.  But, sure, I can add newlines.